webapp/src/store/boards.ts

```diff
@@ -72,6 +72,7 @@ export const updateMembersEnsuringBoardsAndUsers = createAsyncThunk(
             /* eslint-enable no-await-in-loop */
 
             thunkAPI.dispatch(updateBoards(boardsToUpdate))
+            thunkAPI.dispatch(updateMembers(members))
```

Comment:
what's the reason for this addition?

Reply:
To update board-memberships in redux store. We want it to stay up to date with any WS event.

Reply:
Anything specific you're looking for?

Reply:
I'm not sure if this is needed.

Reply:
I was wondering because I was having a hard time understanding where it fit into the PR and the addition of `addMyBoardMemberships`—was it causing a bug or edge case in state before?

Reply:
That's right, no need to dispatch `updateMembers`. Removed now.

I had to create `addMyBoardMemberships` because the existing store function that updates memberships for the current user (`updateMembersHandler`) assumes the user-board membership to already exist in the DB, with won't if the user is just being added to the board. The assumption of the existence of the same entry in store is used to check if the BoardMember payload deals with the current user.

Here's a snipped of the `updateMembersHandler()` function I'm talking about -

```
for (const member of action.payload) {
        if (state.myBoardMemberships[member.boardId] && state.myBoardMemberships[member.boardId].userId === member.userId) {
            state.myBoardMemberships[member.boardId] = member
        }
    }
```
`state.myBoardMemberships[member.boardId]` won't exist when a user is newly added to a board, and nothing works after that. 

Hence, the need for a new store function that exclusively deals with `state.myBoardMemberships` doesn't need to rely on any existing data.

Please let me know if this is still a bit confusing :)

Reply:
> state.myBoardMemberships[member.boardId] won't exist when a user is newly added to a board, and nothing works after that.

Makes sense, thanks for breaking it down! 👍🏻